Library/Homebrew/dev-cmd/test-bot.rb

```diff
@@ -873,6 +873,7 @@ def sanitize_ARGV_and_ENV
 
     ENV["HOMEBREW_DEVELOPER"] = "1"
     ENV["HOMEBREW_SANDBOX"] = "1"
+    ENV["HOMEBREW_RUBY_MACHO"] = "1"
```

Comment:
I just realized that we're no longer compatible with Ruby 1.8.7 that continues to be used on our 10.9 test bot. Because of this, we'll need to guard this as follows:

```ruby
ENV["HOMEBREW_RUBY_MACHO"] = "1" if RUBY_VERSION.split(".").first.to_i >= 2
```

(We can remove this trailing guard clause once Homebrew finally drops support for older Ruby versions or we restore Ruby 1.8.7 compatibility in `ruby-macho`.)